index.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
+  <meta http-equiv="Pragma" content="no-cache" />
+  <meta http-equiv="Expires" content="0" />
+  <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <script lang="js">
+      window.okapi = {
+        url: 'https://folio-etesting-snapshot-kong.ci.folio.org',
+        authnUrl: 'https://folio-etesting-snapshot-keycloak.ci.folio.org',
+      };
+      window.config = {
```

Comment:
Should we consolidate under, e.g. `window.stripes` or `window.folio`? IDK, I'm just thinking: whatever convention we establish here at the very root of things will be hard to change later on. My gut is keep the footprint here as small as possible and use a single namespace. 

Alternatively, could/should this code look sth like this instead?
```
const okapi = { ... };
const config = { ... };
const hub = new StripesHub(okapi, config);
hub.validateSession(); 
```
i.e. move the initialization call here from `index.js` allowing us to avoid touching `window` at all? Is "validate" the right name, or should it be "initialize" or "bootstrap" or "validateorinitializeorupsertohgodihateupsert"?

Reply:
Agree. I implemented your suggestion and pass the values in through the constructor.